content/finalist/ttjc@Lakshay-Batra/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Lakshay Batra
+type: "finalist"
+name: "Lakshay"
+portfolio: "https://batralakshay.netlify.app/"
+github-link: "https://github.com/Lakshay-Batra"
+linkedin-link: "https://www.linkedin.com/in/lakshay-batra-888726189/"
+looking-for: "job"
+description: "Bootstrap | Material Ui | Javascript | React | jQuery | Node | Express | MongoDB | SQL | RESTful Api | Android App Development "
+---
+
+Hi! Looking for a MERN web developer? I am a pre final year CSE student and a MERN web developer, let me walk you through my intro, projects and blogs.
+
+### introduction
+
+I am a person who looks around every perspective of a work and its result before doing it. I am passionate about my work and highly organized. I always keep the end goal in my mind and work always in favour of it. I search for new and effective ways which can lead me to my goal. I am career oriented and hardworking guy.
+
+## why are you taking this challenge?
+
+I need a Job as soon as I complete my degree next year in June 2021 or an internship before that.
+I want to learn and grow fast which this challenge can help me with.
+
+## something interesting about you?
+
+I am good with cooking.
+
+### help I need?
+
+It would be good if someone tells me what all other things i should learn after completing MERN development.
+
+### how to contact me?
+
+- [Twitter](https://twitter.com/Lakshay41026635)
+- [LinkedIn](https://www.linkedin.com/in/lakshay-batra-888726189/)
+
+### projects
+
+not a lot of projects are done. I will keep updating it with real links when done.
```

Comment:
@Lakshay-Batra can you remove this "not a lot......" also check if it's there inside your participant's file